src/json_schema/vfb_query.json

```diff
@@ -12,6 +12,9 @@
     "dataset": {
       "$ref": "mod.json#definitions/minimal_entity_info"
     },
+    "cluster": {
+      "$ref": "mod.json#definitions/minimal_entity_info"
+    },
```

Comment:
Added cluster as permitted key in results json, + value= standard minimal entity info